wca-regulations.md

```diff
@@ -252,8 +252,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 9f13) For "Mean of 3" and "Average of 5" rounds, rankings are assessed based on the ordering of the averages/means of the competitors, where "better" is the shorter recorded time.
     - 9f14) For "Mean of 3" and "Average of 5" rounds, if two or more competitors achieve identical average/mean results, rankings are assessed based on the best attempt per competitor, where "better" is defined as the shorter time.
     - 9f15) Competitors who achieve the same result in a round receive an identical ranking for the round.
-- 9g) A Combined Round consists of two phases of attempts, where competitors advance to the second phase if they meet a designated cutoff during the first phase.
-    - 9g2) Whether a competitor proceeds to next phase of a Combined round must be decided by result (all competitors with a better result than x) of the first phase (see [Regulation 9f12](regulations:regulation:9f12)).
+- 9g) If the round format is "Average of 5", a round may have a "Best of 2" cutoff phase where the competitor must meet a given cutoff with either of the first two solves (e.g. a result better than X seconds) in order to be eligible for the remaining three attempts.
```

Comment:
Hmm, this wording is awkward to me.  The term "phase" seems unnecessary.

How about...
9g) A round may have a designated cutoff, where competitors must meet the cutoff time with one of their first solves in order to be eligible for for the remaining attempts.
9g1) If the round format is "Average of 5", the competitor must meet the cutoff time with one of their first two solves.
9g2) For other round formats, the competitor must meet the cutoff time either with their first solve or with one of their first two solves.  This must be announced by the organization team in advance.

I don't like the word "this" in 9g2, but thoughts?

Reply:
We recently did a pass over the regulations to make sure there aren't any oversights with regards to multibld and fmc. Using the phrase "cutoff time" would break this for multibld (something that I believe was used at worlds in France recently).

Otherwise, I definitely like getting rid of the concept of "phases". I know many people like to think of cutoffs in this way, but I personally think it's a confusing concept, and is not the way I explain the idea to newcomers.

Reply:
Good point.  How about "cutoff result" instead?

Reply:
That sounds better. I'm not sure if we define what it means to "meet" a result anywhere, though. I don't have time to check the regulations, but I think we do define what a "better" result is, so it might be better to use the word "better" here rather than "meet".

Reply:
Yes please! "Better" instead of "meet", "result" instead of "time".